CONTRIBUTING.md

```diff
@@ -175,10 +175,67 @@ the page to activate debug mode. The state of debug mode will be remembered by
 `window.sessionStorage` and will not survive a browser restart or opening the
 docs in a new tab.
 
-### Invisible index anchors
-
-You can create index entries and invisible anchors with `X<|thing,category>`.
-To make them visible activate debug mode.
+### Search indexing rules
+
+Bits of the content can be indexed for quick navigation.
+
+An index item is created using the Pod formatting code `X` in those formats:
+
+    # the `text` is rendered, an item with text `item` under the category `category`
+    # is added to the index
```

Comment:
Why are you using code formatting here? Just use normal text... No need to format this as code comments.

Reply:
My motivation is that plain "the category category" is confusing. Maybe just using quotes would be better though. No real reason, I'm just used to using the code quoting to highlight things.

Reply:
```
    # the "text" is rendered, an item with text "item" under the category "category"
    # is added to the index
    X<text|category,item>
```

Still sounds bad. :(
Can you suggest something more native maybe?

Reply:
The whole thing is formatted as code, due to the 4-space indentation. Those inline-code markings were actually OK (if it had not been formatted as a code comment, that is)

Reply:
Ah, you mean the whole code block.

Well, I just imitated the common way to demonstrate some code, where inline comments nearby are tied to the rest of the code, IMO it's easier to paid attention to anything written this way rather than when plain text is used above (this is often skipped to read the actual code).

Reply:
I'm afraid it's not. Plus you can't use markdown in them, such as the inline-code marks you have actually used. This is also a guideline to documentation contributors, it might not even be clear for them that `#` is a comment. TBH, I though it was a item-number markdown mark the first time I saw it.

Reply:
Addressed with different format.